src/Sulu/Bundle/AdminBundle/Resources/js/views/List/List.js

```diff
@@ -361,10 +361,13 @@ class List extends React.Component<Props> {
                         adapters,
                         addView,
                         editView,
+                        filterable,
                         itemDisabledCondition,
                         searchable,
                         selectable,
                         paginated,
+                        adapterOptions,
+                        showColumnOptions,
```

Comment:
i think this will hide the options per default at the moment, right?

Reply:
no, the default value for this property is `true` https://github.com/Prokyonn/sulu/blob/feature/activity-list/src/Sulu/Bundle/AdminBundle/Resources/js/containers/List/List.js#L79